cmd/appregistry-server/main.go

```diff
@@ -89,17 +89,20 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 
 	loader, err := appregistry.NewLoader(kubeconfig, dbName, downloadPath, logger)
 	if err != nil {
-		logger.Fatalf("error initializing - %v", err)
+		logger.Fatalf("error initializing: %s", err)
 	}
 
 	store, err := loader.Load(sources, packages)
 	if err != nil {
-		logger.Fatalf("error loading manifest from remote registry - %v", err)
+		logger.WithError(err).Warn("error loading app registry manifests")
+	}
+	if store == nil {
+		logger.Fatal("store failed to initialize")
```

Comment:
If there are no bundles that are valid this will crash loop correct? Wondering if it makes sense to expose an empty catalog instead?

Reply:
If there are no bundles the store will not be `nil`. My previous PR exposes an "Empty" `Query` implementation, but  this will only be nil when something is *really* wrong. If you think it's best I can pull in that empty implementation for this PR too.

Reply:
I added the empty querier just to be safe.